src/store.js

```diff
@@ -24,7 +24,7 @@ class Store {
       paymentsResponse: null,
       peersResponse: null,
       channelsResponse: null,
-
+      paymentRequestResponse: observable({}),
```

Comment:
Are you sure we need to declare a separate observable here? The store instance `this` is already declared as an observable [here](https://github.com/lightninglabs/lightning-app/pull/175/files#diff-3bf35cd41da51153dcca3003656f7c67R10).

Reply:
Nope! Good catch, I removed it. I'm kind of wrapping my head around observables / mobx as we go here, so appreciate you pointing out things like this! I wrapped it in `observable` as the property below it was, however I tested it out and still worked as expected after I removed it.